client/src/components/layout/footer/footer.tsx

```diff
@@ -0,0 +1,90 @@
+import { mobilePaths } from "@/lib/paths";
+import Link from "next/link";
+import { Copyright } from "@/components/layout/footer/copywright";
+import { Dot, ExternalLink, Linkedin } from "lucide-react";
+import { BlueskyLogoIcon } from "@/assets/icons/bluesky-logo-icon";
+
+export function Footer() {
+  return (
+    <footer
+      className={"flex flex-col items-center justify-center bg-white px-4 pt-14 pb-2 md:px-0"}
+    >
+      <div className={"container flex flex-col"}>
+        <div className={"flex flex-col gap-2"}>
+          <p
+            className={
+              "font-display text-center text-2xl leading-10 font-bold text-stone-700 not-italic"
+            }
+          >
+            Scenario Compass Platform
+          </p>
+          <p className={"text-center font-sans text-xl leading-6 font-normal text-stone-700"}>
+            by IIASA
+          </p>
+        </div>
+
+        <div className="flex flex-col items-center justify-center gap-7 pt-16 pb-12 text-center md:flex-row md:gap-14 md:pb-16">
+          {mobilePaths.map((path) => {
+            return (
+              <Link
+                key={path.label}
+                href={path.href}
+                className="font-sans text-base leading-6 font-normal text-stone-700 hover:opacity-60"
+              >
+                {path.label}
+              </Link>
+            );
+          })}
+        </div>
+
+        <div
+          className={
+            "flex flex-col items-center justify-between gap-6 border-t border-stone-200 py-14 md:flex-row md:gap-0 md:py-6"
+          }
+        >
+          <div className="order-3 flex flex-col items-center justify-center gap-6 md:order-1 md:flex-row md:gap-4">
+            <Copyright />
+            <Dot className="text-stone-500" />
+            <div className="flex gap-9 md:gap-4">
+              <Link
+                href={""}
+                className="text-right font-sans text-sm leading-5 font-normal text-stone-500"
+              >
+                Privacy Policy
+              </Link>
+              <Link
+                href={""}
+                className="text-right font-sans text-sm leading-5 font-normal text-stone-500"
+              >
+                Cookie Policy
+              </Link>
+            </div>
+          </div>
+          <Dot className="order-2 text-stone-500 md:hidden" />
+          <div
+            className={
+              "order-1 flex flex-col items-center justify-center gap-5 md:order-2 md:flex-row"
+            }
+          >
+            <Link
+              href={"https://shape.apps.ece.iiasa.ac.at/explorer"}
+              target="_blank"
```

Comment:
Whenever we point to an external link, let's add `rel="nofollow noopener"`


https://www.elegantthemes.com/blog/wordpress/rel-noopener-noreferrer-nofollow#what-is-relnoopener-noreferrer